discovery-provider/src/api/v1/playlists.py

```diff
@@ -269,7 +274,9 @@ def get(self, playlist_id):
 trending_response = make_response("trending_playlists_response", ns, fields.List(fields.Nested(playlist_model)))
 trending_parser = reqparse.RequestParser()
 trending_parser.add_argument('time', required=False)
-@ns.route("/trending")
+
+@ns.route("/trending/", defaults={"version": TrendingVersion.DEFAULT.name})
```

Comment:
https://discovery1-prod.sandbox.audius.co/v1/full/tracks/trending redirects to https://discovery1-prod.sandbox.audius.co/v1/full/tracks/trending/ (http 308)

I'm slightly worried this may cause some unintended consequences for third parties consuming the API and it's slightly less performant since we would need to hit two endpoints. do we need the trailing slash?

Reply:
will look for another way of doing this without the redirection, or will separate the endpoints for now. thx